src/screens/index.tsx

```diff
@@ -8,6 +8,8 @@ import { colors } from '@app/shared/lib';
 import { RootStackParamList } from '@shared/lib/navigation';
 import { Text, CloseIcon } from '@shared/ui';
 
+import packageJson from '../../package.json';
```

Comment:
Please create a constant value in shared/lib/constants.tsx that will keep package.json's version and use it here

Reply:
ok